support-frontend/assets/components/priceCards/priceCardsContainer.tsx

```diff
@@ -33,43 +42,67 @@ const contributionTypeToPaymentInterval: Partial<
 };
 
 export function PriceCardsContainer({
-	frequency,
+	paymentFrequency,
 	renderPriceCards,
 }: PriceCardsContainerProps): JSX.Element {
 	const dispatch = useContributionsDispatch();
-	const currency = useContributionsSelector(
-		(state) => state.common.internationalisation.currencyId,
+	const { countryGroupId, currencyId } = useContributionsSelector(
+		(state) => state.common.internationalisation,
 	);
 	const { amounts } = useContributionsSelector((state) => state.common);
 	const { amountsCardData } = amounts;
 	const { selectedAmounts, otherAmounts } = useContributionsSelector(
 		(state) => state.page.checkoutForm.product,
 	);
 	const minAmount = useContributionsSelector(getMinimumContributionAmount());
+
+	const inThreeTierVariantB = inThreeTierV2VariantB(
+		useContributionsSelector((state) => state.common).abParticipations,
+	);
+	const tierCards = inThreeTierVariantB ? tierCardsVariantB : tierCardsVariantA;
```

Comment:
I was a little confused why we assign `tierCards` the value of `tierCardsVariantA` here when `inThreeTierVariantB` is `false`, as looking below I see we dom't use `tierCards` if this is the case - so the value of `tierCardsVariantA` will never be used, so why  assign it to `tierCards`? I think we could simplify things here.

Reply:
Ah yes, 'tierCards' will always be tierCardsVariantB`, have simplified `tierCards` to `tierCardsVaraiantB`